### DISCUSSION TITLE
NotionAPI 问题

### DISCUSSION THREAD

**Discussion by happyang1** [tangly1024/NotionNext#234] [General]
NotionAPI collectionQuery error NOTION_PAGE_ID Response code 403 (Forbidden), 网页上没有内容了，怎么处理？谢谢！

**Comment by tangly1024** [comment]
> NotionAPI collectionQuery error NOTION_PAGE_ID Response code 403 (Forbidden)

这个提示的意思是，没有权限访问你的Notion数据；应该是你没有将页面公开分享导致的。


解决步骤：
1. 点击右上角的Share，并选择share to the web
<img width="667" alt="图片" src="https://user-images.githubusercontent.com/15920488/169729661-e656cf66-69de-473f-8e36-a46d5d8ee8e0.png">


2. 另外，如果你不希望公开你的notion页面信息，也可以[参考此方式](https://github.com/tangly1024/NotionNext/issues/125#issuecomment-1084013216)，将你的私人token配置在项目中。
